device/nokia/x86_64-nokia_ixr7250e_36x400g-r0/platform_reboot

```diff
@@ -17,6 +17,10 @@ update_reboot_cause()
     sync
 }
 
+echo "Disable all SFPs"
+python3 -c 'import sonic_platform.platform; platform_chassis = sonic_platform.platform.Platform().get_chassis(); platform_chassis.tx_disable_all_sfps()'
+sleep 3
```

Comment:
Any additional delay or so required for 400G/ZR optics if all 36 ports are 400G optics

Reply:
I tested both 36 ports 400G and100G, for disableing TX,  there is not much difference for 100G and 400G.